packages/analytics-browser/src/browser-client.ts

```diff
@@ -83,10 +84,17 @@ export class AmplitudeBrowser extends AmplitudeCore implements BrowserClient {
 
     // Step 3: Set session ID
     // Priority 1: `options.sessionId`
-    // Priority 2: last known sessionId from user identity storage
+    // Priority 2: sessionId from url
+    // Priority 3: last known sessionId from user identity storage
     // Default: `Date.now()`
     // Session ID is handled differently than device ID and user ID due to session events
-    this.setSessionId(options.sessionId ?? this.config.sessionId ?? Date.now());
+    const queryParams = getQueryParams();
+    this.setSessionId(
+      options.sessionId ??
+        (queryParams.sessionId ? Number(queryParams.sessionId) : undefined) ??
```

Comment:
sessionId is a fairly generic name - is it possible that we may pick up unrelated sessionId parameters?

Reply:
This would be good to take to team sync - deviceId probably needs to be less generic as well to reduce the chance of conflicts. An `amplitude` prefix on all of them would work, though we might want to go with something a bit shorter as urls have something like a 2083 character limit on chrome / ie.

Reply:
I remembered Zoom? or other customer has the issue, because, they use deviceId for internal purposes([security limitation](https://amplitude.slack.com/archives/C070W7YG9RR/p1714151781380809)). And, they are using GTM, so they set `amp_device_id=xxx` and call `setDeviceId` in the GTM tag.

Reply:
There are customers that use `deviceId` url query parameter already. So we should not change that. For consistency, can we use `sessionId` query parameter and provide an option to override these default query params with custom ones?

Reply:
@izaaz I'm not very into custom sessionId query parameter as it introduces an extra configuration.

Reply:
What about supporting `sessionId` and `ampSessionId`, and giving `ampSessionId` precedence?

This would allow us to keep the existing naming but have a workaround in case of naming conflicts.

Reply:
@justin-fiedler I like `ampSessionId` but Chris has a good point of using `sessionId` might catch unwanted session Id parameter because it's generic.

Reply:
@Mercy811 I was suggesting that, if set, we always use the "namespaced" session ID first (e.g. `ampSessionId`) and ignore `sessionId`. If `ampSessionId` is not set then we fallback to `sessionId`.

Reply:
What about allowing `ampSessionId=` or `ampSessionId=-1` or something similar to allow preventing reading the `sessionId` if needed?

Reply:
After the call, I get that we haven't supported `sessionId` before, in that cause I would say we name it `ampSessionId` and also add `ampDeviceId` (in addition to `deviceId`) as described [here](https://github.com/amplitude/Amplitude-TypeScript/pull/753#discussion_r1608762357). This way are docs can be clear that query params have a `amp` prefix, but existing customers can continue to use `deviceId`

Reply:
Thanks team for discussion. I added the logic to use `ampDeviceId` and falls back to `deviceId` at https://github.com/amplitude/Amplitude-TypeScript/pull/753/commits/368f4cd0f9a4fc6821b90c0f88526e83d8d30bf4